src/Build/Instance/ProjectInstance.cs

```diff
@@ -1893,6 +1919,12 @@ private void TranslateAllState(ITranslator translator)
             translator.Translate(ref _isImmutable);
             translator.Translate(ref _evaluationId);
 
+            translator.Translate(ref _imports);
```

Comment:
Would you mind fixing the comment for [Translate](https://github.com/microsoft/msbuild/blob/master/src/Build/Instance/ProjectInstance.cs#L1859)?